src/twosat.rs

```diff
@@ -1 +1,109 @@
+use crate::internal_scc;
 
+pub struct TwoSAT {
```

Comment:
I **think** we should name this `TwoSat` rather than `TwoSAT`.

[1. Name](https://rust-lang.github.io/api-guidelines/naming.html)

> In `UpperCamelCase`, acronyms and contractions of compound words count as one word: use `Uuid` rather than `UUID`, `Usize` rather than `USize` or `Stdin` rather than `StdIn`.

Reply:
Thank you.
I fixed it.